Sources/PaymentButtons/CardButton.swift

```diff
@@ -0,0 +1,131 @@
+import UIKit
+import SwiftUI
+
+/// Configuration for Card button
+public final class CardButton: PaymentButton {
+
+    /// Available colors for CardButton.
+    public enum Color: String {
+        case gold
+        case white
+        case black
+        case silver
+        case blue
+
+        var color: PaymentButtonColor {
+            PaymentButtonColor(rawValue: rawValue) ?? .black
+        }
+    }
+
+    /// Available labels for CardButton.
+    public enum Label: String {
+        /// Display no label
+        case none
+
+        /// Display "Pay with" on the left side of the button's logo
+        case payWith = "Pay with"
+
+        /// Display "Debit or Credit Card" on the left side of the button's logo
+        case card = "Debit or Credit Card"
+
+        var label: PaymentButtonLabel? {
+            PaymentButtonLabel(rawValue: rawValue)
+        }
+    }
```

Comment:
We can remove every other option for the label and keep "Debit or Credit Card". BCDC doesn't support the other labels.